sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -7892,9 +7904,12 @@ class SQLConf extends Serializable with Logging with SqlApiConf {
   def legacyXMLParserEnabled: Boolean =
     getConf(SQLConf.LEGACY_XML_PARSER_ENABLED)
 
-  def coerceMergeNestedTypes: Boolean =
+  def mergeCoerceNestedTypes: Boolean =
```

Comment:
Oh, you also knew that the naming is weird, don't you, @szehon-ho ? That's the reason you renaming this in this PR.

Reply:
ah yes, its not strictly related, but realized its better they align.